[makefile] on osx, prefix ignored during 'make install'

I've checked out the 0.90.0 tag into a local branch. I run the following:

`prefix='/opt/draeath/crawl' DATADIR='share/crawl' SAVEDIR='~/.crawl' V='1' make -j5`

Compilation seems happy, and (as expected) it won't run as the data files are missing when you try to run from the build directory. However, when I then try to install (as such):

`prefix='/opt/draeath/crawl' DATADIR='share/crawl' SAVEDIR='~/.crawl' V='1' make -j5 install`

I receive this output:

    Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
    /Applications/Xcode.app/Contents/Developer/usr/bin/make -C rltiles all ARCH=x86_64-apple-darwin15.6.0 TILES=
    Neither DESTDIR nor prefix defined -- nowhere to install to, aborting.
    make: *** [install-data] Error 1

I'm not sure where the indicated --prefix is coming from, and I'm not sure why it thinks prefix hasn't been defined - because it *was.* I've also tried dropping -j5 just in case, and found that made no difference. Also, I should note that I have ownership and permissions on /opt/draeath, if that might matter.

--- a/crawl-ref/source/Makefile
+++ b/crawl-ref/source/Makefile
@@ -370,9 +370,6 @@ MCHMOD_LOGS := 664
 INSTALL_UGRP := games:games
 endif
 
-chroot_prefix :=
-prefix        :=
-
 ifeq ($(patsubst %/local,%,$(patsubst %/,%,$(prefix))),/usr)
 FHS := yes
 endif

